src/main/java/pacApp/pacSoapConnector/context/SoapMarshaller.java

```diff
@@ -0,0 +1,26 @@
+package pacApp.pacSoapConnector.context;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.oxm.jaxb.Jaxb2Marshaller;
+
+import pacApp.pacSoapConnector.SoapConvertCurrencyConnector;
+
+@Configuration
+public class SoapMarshaller {
+	  @Bean
+	  public Jaxb2Marshaller marshaller() {
+	    Jaxb2Marshaller marshaller = new Jaxb2Marshaller();
+	    marshaller.setContextPath("com.consumingwebservice.wsdl");
+	    return marshaller;
+	  }
+
+	  @Bean
+	  public SoapConvertCurrencyConnector cClient(Jaxb2Marshaller marshaller) {
+		SoapConvertCurrencyConnector client = new SoapConvertCurrencyConnector();
+	    client.setDefaultUri("http://localhost:50923/Service1.svc?wsdl");
```

Comment:
Please change to Production URL

Reply:
see above! I will change all URLs if the SOAP Server Version run on cloud

Reply:
They are live since I wrote this. Please change

Reply:
The Wsdl has changed.

Reply:
I add the soap auth functionality.

Reply:
The WSDL will be taken from the "production" currency converter deployment (currently dev branch). The service has auto-deploy for this branch. If you changed something, which there should have been a merge request for, than it already auto deployed a new version to the cloud --> you can change it from localhost.